### DISCUSSION TITLE
Add Support for EventListeners

### DISCUSSION THREAD

**Discussion by TheHuman1** [iddan/react-native-canvas#272] [Q&A]
I would really support adding in `EventListeners` for both mouse and touch. [Link to MDN (mouseover example)](https://developer.mozilla.org/en-US/docs/Web/API/Element/mousemove_event)

**Comment by iddan** [comment]
You can use a gesture respond component on top of the canvas: https://reactnative.dev/docs/gesture-responder-system

**Comment by serobalino** [comment]
how
